students/Rumyantseva/src/components/Layout.jsx

```diff
@@ -1,21 +1,77 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import Header from './Header.jsx';
 import ChartList from './ChatList.jsx'
 import MessageField from './MessageField.jsx';
 
 export default class Layout extends React.Component {
+
+    static propTypes = {
+        chatId: PropTypes.string,
+        chats: PropTypes.object.isRequired,
+    };
+
+    state = {
+        chats: {
+            1: {
+                title: 'Чат 1',
+                //messageList: [1, 3]
+            },
+            2: {
+                title: 'Чат 2',
+                //messageList: [2]
+            },
+        },
+        // messages: {
+        //     1: {
+        //         author: 'bot',
+        //         text: 'Привет!'
+        //     },
+        //     2: {
+        //         author: 'bot',
+        //         text: 'Hello!'
+        //     },
+        //     3: {
+        //         author: 'bot',
+        //         text: 'Как дела?'
+        //     },
+        // },
+    }
+
+
+
+    handleAddChat = (title) => {
+        this.setState((state) => {
+            const chatId = Object.keys(state.chats).lenght + 1;
```

Comment:
```suggestion
            const chatId = Object.keys(state.chats).length + 1;
```
опечатка, из-за которой не работает добавление